<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioExpansionSettingsDialog</class>
 <widget class="QDialog" name="AudioExpansionSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Expansion Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Circular Wrap:</string>
     </property>
     <property name="buddy">
      <cstring>circularWrap</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="circularWrap">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>90</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="circularWrapLabel">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Shift:</string>
     </property>
     <property name="buddy">
      <cstring>shift</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSlider" name="shift">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="shiftLabel">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Depth:</string>
     </property>
     <property name="buddy">
      <cstring>depth</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSlider" name="depth">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="depthLabel">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Focus:</string>
     </property>
     <property name="buddy">
      <cstring>focus</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QSlider" name="focus">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="focusLabel">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Center Image:</string>
     </property>
     <property name="buddy">
      <cstring>centerImage</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QSlider" name="centerImage">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="centerImageLabel">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Front Separation:</string>
     </property>
     <property name="buddy">
      <cstring>frontSeparation</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QSlider" name="frontSeparation">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frontSeparationLabel">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Rear Separation:</string>
     </property>
     <property name="buddy">
      <cstring>rearSeparation</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QSlider" name="rearSeparation">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rearSeparationLabel">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Low Cutoff:</string>
     </property>
     <property name="buddy">
      <cstring>lowCutoff</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QSlider" name="lowCutoff">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lowCutoffLabel">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>High Cutoff:</string>
     </property>
     <property name="buddy">
      <cstring>highCutoff</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QSlider" name="highCutoff">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="highCutoffLabel">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close|QDialogButtonBox::StandardButton::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="icon">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Audio Expansion Settings&lt;/span&gt;&lt;br/&gt;These settings fine-tune the behavior of the FreeSurround-based channel expander.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Block Size:</string>
     </property>
     <property name="buddy">
      <cstring>blockSize</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QSlider" name="blockSize">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>8192</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="pageStep">
        <number>128</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blockSizeLabel">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>blockSize</tabstop>
  <tabstop>circularWrap</tabstop>
  <tabstop>shift</tabstop>
  <tabstop>depth</tabstop>
  <tabstop>focus</tabstop>
  <tabstop>centerImage</tabstop>
  <tabstop>frontSeparation</tabstop>
  <tabstop>rearSeparation</tabstop>
  <tabstop>lowCutoff</tabstop>
  <tabstop>highCutoff</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
